src/main/java/io/confluent/examples/streams/utils/MonitoringInterceptorUtils.java

```diff
@@ -23,22 +23,45 @@
     private MonitoringInterceptorUtils() {
     }
 
-    public static void maybeConfigureInterceptors(final Properties streamsConfig) {
-        try {
-            Class.forName(PRODUCER_INTERCEPTOR);
-        } catch (final ClassNotFoundException e) {
-            System.out.println("Monitoring Producer Interceptors not found, skipping");
-            return;
+    public static void maybeConfigureInterceptorsStreams(final Properties streamsConfig) {
+        if (hasMonitoringConsumerInterceptor() && hasMonitoringProducerInterceptor()) {
```

Comment:
should this be two independent `if` blocks so that we can add the consumer or producer interceptors independently?

Or is that not a thing?

Reply:
From previous comments https://github.com/confluentinc/kafka-streams-examples/pull/135#discussion_r204580802 , I'll leave as is for now and we can change later if we want.